integration-tests/common/src/test/kotlin/me/tatarka/inject/test/VisibilityTest.kt

```diff
@@ -0,0 +1,64 @@
+package me.tatarka.inject.test
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import me.tatarka.inject.annotations.Component
+import org.junit.Test
+import kotlin.reflect.KVisibility
+import kotlin.reflect.full.functions
+import kotlin.reflect.full.memberProperties
+
+@Component
+abstract class VisibilityTestComponent {
+    abstract val myPublicProperty: Foo
```

Comment:
<img width="549" alt="Screen Shot 2020-11-08 at 5 14 26 PM" src="https://user-images.githubusercontent.com/628930/98485650-e64d8700-21e5-11eb-9ef8-36ef5f301cfa.png">
It looks like when overriding you don't need to specify the modifier.

Reply:
I'll remove the visibility modifier from properties when they are generated